<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigObjectListView</class>
 <widget class="QWidget" name="ConfigObjectListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListView" name="listView"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="itemUp">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="edit-move-up">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="itemDown">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="edit-move-down">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
